client/state/selectors/is-site-checklist-complete.js

```diff
@@ -52,6 +52,11 @@ export default function isSiteChecklistComplete( state, siteId ) {
 			return true;
 		}
 
+		// Starting site setup autocompletes, so it shouldn't cause an incomplete checklist.
+		if ( CHECKLIST_KNOWN_TASKS.START_SITE_SETUP === task.id ) {
+			return true;
+		}
+
```

Comment:
A bit of a nit, but before merging could you update this function's JS docs to bring it up to date with the code changes?

Reply:
Done, thanks for pointing that out!